openpype/hosts/nuke/plugins/load/load_gizmo_ip.py

```diff
@@ -28,7 +29,7 @@ class LoadGizmoInputProcess(load.LoaderPlugin):
 
     families = ["gizmo"]
     representations = ["*"]
-    extensions = {"gizmo"}
+    extensions = {"nk"}
```

Comment:
Should we keep both? `nk` and `gizmo`?

Reply:
only `nk` is the extension. `gizmo` is representation name in our publishing schema.